Example/FinancialConnections Example/FinancialConnectionsUITests/FinancialConnectionsUITests.swift

```diff
@@ -391,6 +391,29 @@ final class FinancialConnectionsUITests: XCTestCase {
             // 'cancel' the test as the bank is under the maintenance
         }
     }
+
+    func testWebInstantDebitsFlow() throws {
```

Comment:
might make sense to eventually split these into another UITest class if easy to do